platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/DetailedConsensusEventTest.java

```diff
@@ -60,22 +61,24 @@ public void serializeAndDeserializeConsensusEvent() throws IOException {
 
     @Test
     public void EventImplGetHashTest() {
-        DetailedConsensusEvent consensusEvent = generateConsensusEvent();
-        EventImpl event = new EventImpl(
+        final PlatformContext platformContext =
+                TestPlatformContextBuilder.create().build();
+        final DetailedConsensusEvent consensusEvent = generateConsensusEvent();
+        final EventImpl event = new EventImpl(
                 consensusEvent.getBaseEventHashedData(),
                 consensusEvent.getBaseEventUnhashedData(),
                 consensusEvent.getConsensusData());
-        CryptographyHolder.get().digestSync(consensusEvent);
-        Hash expectedHash = consensusEvent.getHash();
-        CryptographyHolder.get().digestSync(event);
+        platformContext.getCryptography().digestSync(consensusEvent);
+        final Hash expectedHash = consensusEvent.getHash();
+        platformContext.getCryptography().digestSync(event);
```

Comment:
It is not clear IMO against what we are testing here, if `Cryptography` is taken from something called `TestPlatformContextBuilder`? Even before your changes, I think it is questionable to compare two calculated values in a unit-test.

Reply:
Can you create an issue for that?